code/modules/organs/internal/_internal.dm

```diff
@@ -13,11 +13,11 @@
 	var/damage_reduction = 0.5     //modifier for internal organ injury
 
 /obj/item/organ/internal/Initialize(mapload, material_key, datum/dna/given_dna)
+	. = ..()
```

Comment:
Dunno if you tried it out, but you probably want to leave the parent call after, since icon_updates are going to get triggered with the alive icon not set in some organ's case.

Reply:
Its probably worth just moving those to set_species though.

Reply:
Damage threshold set turned out to be redundant, it's done in set_max_damage() already. Moved parent call back to the end.